README.md

```diff
@@ -39,7 +39,6 @@ This applies to the SPIKE Essential small hub.
 |            **Gyroscope** | Three-axis gyroscope (IC: ?).                                                    |
 |                **Ports** | 2 LPF2 ports = 2 normal speed (115kB).                                           |
 
-
```

Comment:
Could you remove this change from the MR?

Reply:
The blank line became a bullet point?

Reply:
Sorry, I deleted that bullet point.